update_bash_env.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+"""
+Generate a file for use by BASH_ENV
+"""
+
+import json
+from pathlib import Path
+from logging import getLogger, basicConfig
+from argparse import ArgumentParser
+from yaml import safe_load, YAMLError
+from jinja2 import Environment, FileSystemLoader
+
+formatting = json.dumps(
+    {
+        "timestamp": "%(asctime)s",
+        "namespace": "%(name)s",
+        "loglevel": "%(levelname)s",
+        "message": "%(message)s",
+    }
+)
+basicConfig(level="WARNING", format=formatting)
+LOG = getLogger("easy_infra")
+
+
+def render(*, template_file: Path, config: dict, output_file: Path) -> None:
+    """Render the functions file"""
+    folder = str(template_file.parent)
+    file = str(template_file.name)
+    template = Environment(loader=FileSystemLoader(folder)).get_template(file)
+    out = template.render(config)
+    output_file.write_text(out)
+    output_file.chmod(0o755)
+
+
+def create_arg_parser() -> ArgumentParser:
+    """Parse the arguments"""
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--config-file",
+        type=lambda p: Path(p).absolute(),
+        default=Path("easy_infra.yml").absolute(),
+        help="specify a config file",
+    )
+    parser.add_argument(
+        "--output",
```

Comment:
instead of a lambda, i would use a function and validate all these paths for exists etc.  then raise an exception maybe?

Reply:
this is a preference I guess, you are just letting it fail later when you are using it

Reply:
Yeah, opportunity to improve in the future